includes/Media.php

```diff
@@ -238,38 +253,16 @@ public static function rest_api_init() {
 					'enum'        => [ 'editor' ],
 					'context'     => [ 'view', 'edit', 'embed' ],
 				],
-				'get_callback'    => static function ( $prepared ) {
-					$id = $prepared['id'];
-
-					$terms = wp_get_object_terms( $id, self::STORY_MEDIA_TAXONOMY );
-					if ( is_array( $terms ) && $terms ) {
-						$term = array_shift( $terms );
-
-						return $term->slug;
-					}
-
-					return '';
-				},
-				'update_callback' => static function ( $value, $object ) {
-					wp_set_object_terms( $object->ID, $value, self::STORY_MEDIA_TAXONOMY );
-				},
+				'get_callback'    => [$this, 'media_source_get_callback'],
```

Comment:
Remove static functions and remove with method. This way in our tests, we can now write unit tests.